src/connector/src/source/common.rs

```diff
@@ -82,7 +82,7 @@ pub(crate) async fn into_chunk_stream(
                     .flat_map(|msg| msg.payload.as_ref().map(|p| p.len() as u64))
                     .sum();
 
-                partition_input_count
+                partition_bytes_count
```

Comment:
No wonder I found the surprising counts 

<img width="928" alt="image" src="https://github.com/user-attachments/assets/9c8ced93-36f5-4488-90e1-434e9df0ab88">